extensions/wikia/Search/classes/Services/EntitySearchService.php

```diff
@@ -194,7 +194,7 @@ public function setHubs( $hubs ) {
 	}
 
 
-	protected function prepareQuery( $query ) {
+	protected function prepareQuery( string $query ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$query". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)